_examples/golang-basics/main.go

```diff
@@ -28,6 +28,18 @@ func startServer() error {
 	})
 
 	webrpcHandler := NewExampleServiceServer(&ExampleServiceRPC{})
+	webrpcHandler.OnRequest = func(w http.ResponseWriter, r *http.Request) {
```

Comment:
can we return error from this hook, so we can halt the handler execution?

(e.g. to respond with HTTP 403 on Permission Denied based on some ACL annotation)